src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -213,6 +209,13 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                         package.PushTriangleVertex(d.X, d.Y, d.Z);
                         package.PushTriangleVertex(a.X, a.Y, a.Z);
 
+                        package.PushTriangleVertexUV(0,0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+
```

Comment:
Because there are UVs, they're just all 0,0. In order to flip that bit, I would have to edit the `GeometryModel3D` base class inside Helix which expects that you have an equivalent number of texture coordinates for the vertices supplied, OR, I'd have to inject this data right before we render.